File: C#JAVA_Client_Server/ClientServcerC#Protocols/LongExample/Domain/src/main/java/entities/domain/Utilizator.java
package entities.domain;

public class Utilizator extends Entitate<Integer> {
    String numeUtilizator;
    String passWord;

    @Override
    public String toString() {
        return "entities.domain.Utilizator{" +
                "numeUtilizator='" + numeUtilizator + '\'' +
                '}';
    }

    public Utilizator() {
    }

    public Utilizator(String numeUtilizator, String passWord) {
        this.numeUtilizator = numeUtilizator;
        this.passWord = passWord;
    }

    public Utilizator(String numeUtilizator) {
        this.numeUtilizator = numeUtilizator;
    }

    public String getNumeUtilizator() {
        return numeUtilizator;
    }

    public void setNumeUtilizator(String numeUtilizator) {
        this.numeUtilizator = numeUtilizator;
    }

    public String getPassWord() {
        return passWord;
    }

    public void setPassWord(String passWord) {
        this.passWord = passWord;
    }
}

File: C#JAVA_Client_Server/ClientServcerC#Protocols/Solution1/Domain/Utilizator.cs
using System;
using Domain;

namespace ConsoleApp1.domain
{
    [Serializable]
    public class Utilizator : IObject<int>
    {
        private String numeUtilizator;
        private string passwd;

        public Utilizator(string numeUtilizator, string passwd)
        {
            this.numeUtilizator = numeUtilizator;
            this.passwd = passwd;
        }

        public Utilizator(string numeUtilizator)
        {
            this.numeUtilizator = numeUtilizator;
        }

        public Utilizator()
        {
            
        }

        public override string ToString()
        {
            return base.ToString()+" "+numeUtilizator;
        }

        public string NumeUtilizator
        {
            get => numeUtilizator;
            set => numeUtilizator = value;
        }

        public int Id { get; set; }
        public string Passwd
        {
            get => passwd;
        }
    }
}